softioc/device.py

```diff
@@ -1,9 +1,9 @@
 import os
+import sys
 import time
-import inspect
-from ctypes import *
 import numpy
 
+from ctypes import *
```

Comment:
I tend to be a bit queasy about `import *`, and I suspect there aren't that many uses of `ctypes` values in this file, so try replacing this with a simple `import ctypes`.

Reply:
There are 14 references to items in that file - I'll replace the import with 
`from ctypes import CFUNCTYPE, c_int, c_void_p, memmove`

Reply:
Didn't realise that was inherited: see the line popped up because you moved it!